src/components/summaryDetail/Nutrition.jsx

```diff
@@ -0,0 +1,29 @@
+import { nanoid } from 'nanoid';
+
+export default function Nutrition({ nutrition }) {
+  let nutritionEls = null;
+
+  function capitalize(word) {
+    return (word.charAt(0).toUpperCase() + word.slice(1));
+  }
+
+  if (Object.keys(nutrition).length === 0) {
+    return nutritionEls = <p text-xs>No nutrition data available</p>;
+  }
+  nutritionEls = Object.entries(nutrition)
+    .filter(([key, value]) => key !== 'updated_at')
+    .sort(([keyA], [keyB]) => keyA.localeCompare(keyB))
+    .map(([key, value]) => (
+      <p key={nanoid()} className="border-t text-xs">
```

Comment:
As Robester mentioned in the code review of my code, "I assume nanoid is a random key generator. It is not recommended to use a random key generator as a key. You should use some sort of combination of strings from the recipes object or a unique id provided by the backend, which should also be in the recipes object."

Reply:
Addressed with commit [6148bb1](https://github.com/chingu-voyages/v46-tier2-team-20/pull/51/commits/6148bb1f4f35ee60f226aa0f13b608da3f5975ad), below